GPU/GPUTracking/Debug/GPUTPCClusterFilter.h

```diff
@@ -28,8 +30,29 @@ namespace o2::gpu
 class GPUTPCClusterFilter
 {
  public:
-  GPUTPCClusterFilter(const o2::tpc::ClusterNativeAccess& clusters);
+  GPUTPCClusterFilter(const o2::tpc::ClusterNativeAccess& clusters, uint8_t filterType);
   bool filter(uint32_t sector, uint32_t row, o2::tpc::ClusterNative& cl);
+
+ private:
+  uint32_t mMaxTimeBin = 14256;
+  uint8_t mFilterType = 0; //< 0: off, 1: custom, 2: PbPb23
+  static constexpr int MaxStacks = GPUCA_NSECTORS * 4;
+
+  std::vector<int> mClusterStats[MaxStacks]; //< Number of clusters per stack and time bin
```

Comment:
This should not be a constant array, since the class is always initialized (even multiple times), also when not used.
Could you make this a unique_ptr to std::vector<int>[] and initialize only if filterType == 2?